DRAFT.md

```diff
@@ -1104,19 +1111,19 @@ object may only become valueless after it has been moved from.
 `allocator_traits<Allocator>::value_type` is not the same type as`T`, the
 program is ill-formed. Every object of type `polymorphic<T, Allocator>` uses an
 object of type `Allocator` to allocate and free storage for the owned object as
-needed. The owned object is constructed using the function
-`allocator_traits<allocator_type>::rebind_traits<U>::construct` and destroyed
- using the function
-`allocator_traits<allocator_type>::rebind_traits<U>::destroy`, where `U` is
-either `allocator_type::value_type` or an internal type used by the polymorphic
-value.
-
-3. Copy constructors for a polymorphic value obtain an allocator by calling
+needed.
+
+3. Constructing an owned object with arguments `args...` using an allocator `a`
+means calling `allocator_traits<allocator_type>::rebind_traits<U>::construct(a,
```

Comment:
Strictly speaking, you don't need to rebind the traits here. The `construct` member is a function template so that `Alloc<T>` can construct+destroy objects of type `U`. So it's up to you whether you want to include the rebind for consistency/simplicity, or omit it (which would mean slightly adjusting the "and U is either ..." text to only refer to the `allocate` call). The `allocate` call below this does need to use the rebound type (and it does so that's good).